src/Resources/skeleton/authenticator/Empty.tpl.php

```diff
@@ -9,7 +9,7 @@
 use Symfony\Component\Security\Core\User\UserProviderInterface;
 use Symfony\Component\Security\Guard\GuardAuthenticator;
 
-class {{ class_name }} extends GuardAuthenticator
+class <?php echo $class_name; ?> extends GuardAuthenticator
```

Comment:
Just asking: would it be acceptable to replace this:

```php
<?php echo $var; ?>
```

by this?

```php
<?= $var ?>
```

The `<?=` syntax is always available since PHP 5.4 and it cannot be disabled in any way.

Reply:
Hmm, I didn't know that. I thought that syntax was somehow deemed "horrible" by someone... but I can't remember. I'm fine with it - we'll see if anyone has a good reason against it.

Reply:
Maybe it's horrible. Not sure. The `<?php echo ...` looked too long to me ... but maybe it's better to use it because it's the normal syntax everybody know and likes.

Reply:
I like the short syntax :) This is PHP, let's use the language's features ;)

Reply:
I like the short syntax as well.

Reply:
I agree about using the short echo syntax